lib/ex_machina/sequence.ex

```diff
@@ -3,6 +3,12 @@ defmodule ExMachina.Sequence do
     Agent.start_link(fn -> HashDict.new end, name: __MODULE__)
   end
 
+  def next(sequence_name) do
```

Comment:
That's a good point. I'm not sure though. Should we raise, or should we convert it to a string? I think raise, but just want to make sure :)